gradle/javaProject.gradle

```diff
@@ -201,12 +202,3 @@ publishing {
 }
 
 test.finalizedBy(jacocoMergeExec)
-//compileJava {
-//    doFirst {
-//        inputs.property("moduleName", moduleName)
-//        options.compilerArgs = [
-//                '--module-path', classpath.asPath,
-//        ]
-//        classpath = files()
-//    }
-//}
```

Comment:
Shall we remove the commented out code?

Reply:
Sure!

Reply:
It is a template how you can fix Java 9 Module errors but it shouldn't be applied to every Java project from my understanding. This configuration is only needed if the module depends on a non-module jar without any "Automatic-Module-Name" in its MANIFEST.MF

Reply:
Yes, if we don't need this here, we can completely remove it rather than commenting-out, right?